code/modules/antagonists/_common/antag_team.dm

```diff
@@ -168,7 +168,8 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 		return
 
 	for(var/datum/mind/M as anything in members)
-		to_chat(M.current, "<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>")
+		SEND_SOUND(M.current, sound('sound/effects/adminticketopen.ogg'))
+		to_chat(M.current, chat_box_red("<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span></font><span class='notice'>[message]</span>"))
```

Comment:
Can you make this into a var before the looping, so it doesnt have to so many string concatenations? Possible the same for the sound() call.